src/RenderedEmailTemplateMailTest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Craftzing\Laravel\NotificationChannels\Postmark;
+
+use PHPUnit\Framework\TestCase;
+
+final class RenderedEmailTemplateMailTest extends TestCase
+{
+    /** @test */
+    public function itCanCreateRenderdEmailTemplate(): void
+    {
+        $subject = 'Some Content Here';
+        $htmlBody = 'Some HTML Content Here';
+        $textBody = 'Some Text Content Here';
+
+        $mail = RenderedEmailTemplateMail::fromRenderedContent($subject, $htmlBody, $textBody);
+
+        $this->assertSame($subject, $mail->subject);
+        $this->assertSame($textBody, $mail->textView);
```

Comment:
I know Laravel doesn't expose a way to publicly access the `html` property on a mailable, but maybe we should add a `htmlBody()` method to the `RenderedEmailTemplateMail` so we can inspect it properly. WDYT?